### ISSUE TITLE
NPP v7.9.4 + SCI v4.4.6: new messages

### ISSUE THREAD

**Issue by pryrt** [pryrt/Win32-Mechanize-NotepadPlusPlus#69]
With the release of v794, which updated Scintilla from v4.2.0 to v4.4.6, there were some new or changed messages and constants.

I will need to add these to the auto-generated or manually generated code.

Doing a diff between the old and new, found the following additions (and one change of value)

```
#define SCI_MARKERHANDLEFROMLINE 2732
#define SCI_MARKERNUMBERFROMLINE 2733

#define SCI_SETTABMINIMUMWIDTH 2724
#define SCI_GETTABMINIMUMWIDTH 2725

#define SCI_SETTARGETSTARTVIRTUALSPACE 2728
#define SCI_GETTARGETSTARTVIRTUALSPACE 2729

#define SCI_SETTARGETENDVIRTUALSPACE 2730
#define SCI_GETTARGETENDVIRTUALSPACE 2731

#define SCI_BRACEMATCHNEXT 2369

#define SCI_GETMULTIEDGECOLUMN 2749

#define SCI_GETSELECTIONNSTARTVIRTUALSPACE 2726

#define SCI_GETSELECTIONNENDVIRTUALSPACE 2727

#define SCI_EOLANNOTATIONSETTEXT 2740
#define SCI_EOLANNOTATIONGETTEXT 2741
#define SCI_EOLANNOTATIONSETSTYLE 2742
#define SCI_EOLANNOTATIONGETSTYLE 2743
#define SCI_EOLANNOTATIONCLEARALL 2744
#define EOLANNOTATION_HIDDEN 0
#define EOLANNOTATION_STANDARD 1
#define EOLANNOTATION_BOXED 2
#define SCI_EOLANNOTATIONSETVISIBLE 2745
#define SCI_EOLANNOTATIONGETVISIBLE 2746
#define SCI_EOLANNOTATIONSETSTYLEOFFSET 2747
#define SCI_EOLANNOTATIONGETSTYLEOFFSET 2748

#define SCI_SETILEXER 4033

#define SC_MOD_CHANGEEOLANNOTATION 0x400000
#define SC_MODEVENTMASKALL 0x7FFFFF             => changed value

```

**Comment by pryrt** [comment]
verified that menuCmdId.h and Notepad_plus_msgs.h didn't change (except for copyright/license text), so nothing new to implement there

**Comment by pryrt** [comment]
closed by #70
